README.md

```diff
@@ -999,6 +999,7 @@ Games, game servers and control panels.
 **[`^        back to top        ^`](#)**
 
 - [BlueMind](https://www.bluemind.net/en/) - Groupware with email, calendar, addressbooks, exchange active sync, exchange MAPI protocol support. ([Source Code](https://forge.bluemind.net/stash/projects/BM/repos/bluemind-public/browse)) `AGPL-3.0` `Java`
+- [Carbonio](https://www.zextras.com/carbonio/) - Email, calendar, collaboration server with Web interface and lots of integrations. ([Source Code](https://github.com/Zextras)) `GPL-2.0/CPAL-1.0` `Java`
```

Comment:
The correct link is https://www.zextras.com/carbonio-community-edition/ (since /carbonio/ is the non-free/hosted version)